### PR TITLE
Portal read + write

### PR DESCRIPTION
**Pull request by AndrewGibson27**
@ashley-hebler Obviously still a work in progress, but feel free to poke around at your convenience to see if you hate any of my code.

### CODE REVIEW

File: static/sass/ds/7-utilities/_bg.scss [hunk 1]
```diff
@@ -9,3 +9,17 @@
...
+  }
+}
+
+.has-bg-twitter {
+  background-color: $color-twitter;
+}
+
+.has-bg-facebook {
+  background-color: $color-facebook;
+}
```
Comment (ashley-hebler) [inline static/sass/ds/7-utilities/_bg.scss]: The good news is this is included in queso, so no need to add. The bad news is I failed to document it. I'll add it to the SCSS comments so it appears in the docs. I don't blame you for not seeing that one!

File: static/sass/ds/6-components/_message.scss [hunk 2]
```diff
@@ -22,7 +23,7 @@
     border: 1px solid $color-white-off;
     border-radius: 50%;
     cursor: pointer;
-    padding: .25rem .45rem;
+    padding: .5rem .5rem;
```
Comment (ashley-hebler) [inline static/sass/ds/6-components/_message.scss]: For some reason this padding setting is looking a little oblong on my screen. I always set a width and height on circle buttons, but maybe that's old school 👵. If you go that route, `l-align-center-children` is helpful to center the little x icon.

Comment (AndrewGibson27) [reply static/sass/ds/6-components/_message.scss]: @ashley-hebler Yeah, I jacked that up. Should be fixed now.

**Comment by ashley-hebler** [comment]
This is so much work and it's looking super slick. I didn't test functionality a whole lot, but that saved state is a nice solve. I like that the loader pauses you for a bit so you interpret the change.

Anyway I left a few notes, but I didn't speak to any of the Vue commits since you are a wizard in that, while I'm a mere muggle ⚡️

File: static/sass/ds/7-utilities/_bg.scss (hunk 1 above)
Comment (AndrewGibson27) [reply static/sass/ds/7-utilities/_bg.scss]: Removed!

**Pull request merged**